index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="./css/style.css">
+</head>
+<body>
+    <header class="header">
+        <div class="header-container">
+            <div class="logo-box header-items">
+                <span>My Logo</span>
+            </div>
+            <nav class="navigation header-items">
+                <img src="./images/Enabled.png" alt="" id="compress-menu">
+            </nav>
+        </div>
+    </header>
+
+    <main class="main">
+        <section class="presentation">
+            <div class="presentation-content">
+                <div class="text-wrapper">
+                    <h2 class="text text-primary">I’m Lawrence <br>Glad to see <br> you!</h2>
```

Comment:
- [ ] Kindly change your header tag to `h1` since this is your first header tag, this will make your page optimized for SEO. Also, ensure your design is responsive it currently misplaced some background images when the width is beo 400px, the minimum recommended is 375px.

Reply:
thank you very much for the review and suggestions. I don't understand  "ensure your design is responsive it currently misplaced some background images when the width is beo 400px, the minimum recommended is 375px" if you can explain much more details for me.
thank you